README.md

```diff
@@ -235,13 +232,13 @@ Having the Infinispan Operator deployed, just jump to [Install Data Index Servic
 
 ### Deploy Strimzi
 
-Deploying [Strimzi](https://strimzi.io/) is easy since it's an Operator and should be available in the [OperatorHub](https://operatorhub.io/operator/strimzi-kafka-operator). On OpenShift Web Console, go to the left menu, Catalog, OperatorHub and search for `Strimzi`.
+Find the [Strimzi](https://strimzi.io/) Operator in the [OperatorHub](https://operatorhub.io/operator/strimzi-kafka-operator). On OpenShift Web Console, go to the left menu, `Operators`, `Installed Operators` and search for `Strimzi`.
```

Comment:
If the OperatorHub is installed, the Kogito Operator will be installed with Strimzi, users shouldn't have to deploy it manually. The only condition to deploy Strimzi manually is on environments that OperatorHub is not available.

Plus, this README should be reviewed after this PR: https://github.com/kiegroup/kogito-cloud-operator/pull/109. That's about to be merged.

Reply:
The "Deploy Strimzi" section should be moved to another section where Kogito Operator is deployed manually. I'd say to have a section named "Deploy Infinispan" with subsections like "Manually" and "via OperatorHub". The same for Strimzi. @sterobin wdyt?

Reply:
Or we have a section in Readme for installation on OpenShift with no OperatorHub (aka mostly OC3) ?

Reply:
Since installing the Kogito operator via OperatorHub also installs Strimzi operator and Infinispan operator, this section actually only describes a simple example of deploying the **custom resources** of Strimzi. I think the details of manually deploying Strimzi and Infinispan operators could be found in their documentation. We could add some links to the documentation in the `Deploying to OpenShift 3.11` section.